CONTRIBUTING.md

```diff
@@ -0,0 +1,65 @@
+# What do I need to know to help?
+
+When contributing to this repository, please feel free to discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+
```

Comment:
Please add a section on "What can you contribute?"
Which would include correcting any errors, missing information, solutions with explanations, links to online available solutions, etc